app/i18n/de/gen.php

```diff
@@ -24,7 +24,7 @@
 		'remove' => 'Entfernen',
 		'rename' => 'Umbenennen',
 		'see_website' => 'Website ansehen',
-		'submit' => 'Abschicken',
+		'submit' => 'Speichern',
```

Comment:
The one place I didn't think of yesterday that's slightly different from the rest is the installation, btw, in case that's relevant.

Reply:
I can't recall the installation UI exactly. 
-> If it is a single page with just a form asking for MySQL Credentials and so on, then "Save" (Speichern) would be ok. 
-> If it is a multi-page setup wizard, then a "Continue"/"Next" Button (Fortfahren/Weiter) would be preferred to get to page 2, page 3, etc.

Reply:
Agreed, it should be changed to continue in parts.